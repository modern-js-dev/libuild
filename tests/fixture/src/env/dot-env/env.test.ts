import { getLibuilderTest } from '@modern-js/libuild-test-toolkit';

describe('fixture:env', () => {
  it('env', async () => {
    const bundler = await getLibuilderTest({
      root: __dirname,
      input: {
        main: './index.ts',
      },
    });
    await bundler.build();
    bundler.expectJSOutputMatchSnapshot();
    delete process.env.NODE_ENV;
  });
});
