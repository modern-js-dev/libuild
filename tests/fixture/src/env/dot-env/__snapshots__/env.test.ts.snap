// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`fixture:env env 1`] = `
"// index.ts
var isNativeRemapping = () => {
  return \\"true\\";
};
var getNodeEnv = () => {
  return \\"development\\";
};
export {
  getNodeEnv,
  isNativeRemapping
};
"
`;
